CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+	}
+
+}
+
+#Rovereta Affair (USA Response)
+country_event = {
+	id = 8020321
+	title = "The Rovereta Affair"
+	desc = EVTDESC8020321
+	picture = "san_marino_nixon"
+	
+	is_triggered_only = yes
+
+	option = {
+		name = "Ask italian troops to retire (War Risk)"
+		random_country = {
+                       limit = { tag = ITA is_vassal = no }
+                       relation = { who = THIS value = -10 }
+        	}
+		random_country = {
+                       limit = { tag = RUS government = proletarian_dictatorship is_vassal = no }
+                       relation = { who = THIS value = 25 }
+        	}
+			
+		ITA = { country_event = 8020322 }
+		ai_chance = { factor = 0.4 }
+	}
+
+	option = {
+		name = "Greenlight the Italian Operation"
+		random_country = {
+                       limit = { tag = ITA is_vassal = no }
+                       relation = { who = THIS value = 35 }
+        	}
+			
+        set_global_flag = San_Marino_Occupied
+		ai_chance = { factor = 0.6 }
+	}
+
+}
+
+#Rovereta Affair (ITA Response)
+country_event = {
+	id = 8020322
+	title = "The Rovereta Affair"
+	desc = EVTDESC8020322
+	picture = "san_marino_nixon"
+	
+	is_triggered_only = yes
+
+	option = {
+		name = "Maintain the occupation regardless"
+		badboy = 5
+		
+		USA = { 
+		    war = { 
+			target = ITA
+			attacker_goal = { casus_belli = territorial_dispute_internal }
+			defender_goal = { casus_belli = territorial_dispute_internal }
+			call_ally = no
+		    }
+		}
+		ai_chance = { factor = 0.2 }
+	}
+
+	option = {
+		name = "Withdraw from San Marino"
+		prestige = -25
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+        743 = { add_province_modifier = { name = generic_uprising duration = 365 } }
+		ai_chance = { factor = 0.8 }
+	}
+
+}
+
+#PCI-PSI Break
+country_event = {
+	id = 8020314
+	title = "PCI-PSI Break"
+	desc = EVTDESC8020314
+	picture = "togliatti_nenni"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1957
+		NOT = { year = 1992 }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		exists = RUS
+		RUS = { government = proletarian_dictatorship }
+		
+		HUN = { OR = { government = proletarian_dictatorship1 government = proletarian_dictatorship } }
+		
+		has_global_flag = hungarian_revolution_1956
+		OR = { has_country_flag = 1953_ITA_Legge_Truffa has_country_flag = 1953_ITA_Centrism }
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Interesting..."
+		
+		any_pop = {
+			scaled_consciousness = { factor = 2 ideology = communist }
+			scaled_consciousness = { factor = 2 ideology = communist_social }
+			scaled_consciousness = { factor = 3 ideology = socialist }
+		}
+		any_pop = { 
+			limit = { strata = middle }
+			ideology = { factor = 0.05 value = socialist }
+			ideology = { factor = -0.05 value = communist }
+		}
+
+	}
+	
+}
+
+#Italian Economic Miracle
```

Comment:
Please remove this event and restore the decision. Flag check here can be used in the decision.